lib/socket/socket_c.nit

```diff
@@ -115,12 +119,16 @@ extern class NativeSocket `{ int* `}
 		return d;
 	`}
 
+	# Free the socket
 	fun destroy `{ free(self); `}
 
+	# Close the socket in both read/write
 	fun close: Int `{ return close(*self); `}
 
+	# Get the FD related to `self`
 	fun descriptor: Int `{ return *self; `}
 
+	# Connect to another open socket
```

Comment:
Shouldn't this comment be about the `addrIn` of another socket?

Like you did with `bind`?

Reply:
I'm not sure what more there is to add here, I've added a SEE: C documentation just in case, but the addrIn structure is already documented, and the `bind` function is already documented in a similar fashion.